Security/LoginManager.php

```diff
@@ -27,21 +27,55 @@
  */
 class LoginManager implements LoginManagerInterface
 {
+    /**
+     *
+     * @var SecurityContextInterface
+     */
     private $securityContext;
+    
+    /**
+     *
+     * @var UserCheckerInterface
+     */
     private $userChecker;
+    
+    /**
+     *
+     * @var SessionAuthenticationStrategyInterface
+     */
     private $sessionStrategy;
+    
+    /**
+     *
+     * @var ContainerInterface
+     */
     private $container;
 
-    public function __construct(SecurityContextInterface $context, UserCheckerInterface $userChecker,
-                                SessionAuthenticationStrategyInterface $sessionStrategy,
-                                ContainerInterface $container)
-    {
+    /**
+     * 
+     * @param SecurityContextInterface $context
+     * @param UserCheckerInterface $userChecker
+     * @param SessionAuthenticationStrategyInterface $sessionStrategy
+     * @param ContainerInterface $container
```

Comment:
This does not add any value compared to the signature as everything is typehinted